Cargo.toml

```
@@ -2,6 +2,15 @@
 name = "CreateProcessW"
 version = "0.1.0"
 edition = "2021"
+license = "MIT or Apache-2.0"
+description = "Create and handle processes on Windows using the Win32 API"
+repository = "https://github.com/yozhgoor/CreateProcessW"
+homepage = "https://github.com/yozhgoor/CreateProcessW"
+documentation = "https://docs.rs/CreateProcessW"
+readme = "README.md"
+keywords = ["windows", "processes", "command", "library"]
```

library? XD

maybe add CreateProccessExW (seems to be a frequent name too)

```suggestion
keywords = ["winapi", "process", "command", "CreateProccessExW"]
```